### ISSUE TITLE
PrivateEndpoint with "privateServiceConnection.isManualConnection: false" errors out

### ISSUE THREAD

**Issue by gdamjan** [hashicorp/cdktf-provider-azurerm#794]
Trying to deploy a stack with a `PrivateEndpoint` that has `privateServiceConnection.isManualConnection` set to `false`, fails to deploy. The generated terraform template doesn't have the `is_manual_connection` field, and then terraform fails to run the deployment, saying it's mandatory.

If I flip the flag to `true`, only then it appears in the generated terraform template.

Example code snippet:
```
new PrivateEndpoint(this, "endpoint", {
  name: "endpoint",
  resourceGroupName: rg.name,
  location: rg.location,
  subnetId: subnet.id,

  privateDnsZoneGroup: {
    name: "privatednszonegroup",
    privateDnsZoneIds: [dns.id],
  },

  privateServiceConnection: {
    name: "appservice-privateconnection",
    privateConnectionResourceId: app.id,
    subresourceNames: ["sites"],
    isManualConnection: false,
  },
});
```

The generated terraform template, doesn't have the `is_manual_connection` field:
```
    "azurerm_private_endpoint": {
      "endpoint": {
        "location": "${azurerm_resource_group.rg.location}",
        "name": "endpoint",
        "resource_group_name": "${azurerm_resource_group.rg.name}",
        "subnet_id": "${azurerm_subnet.subnet.id}",
        "private_dns_zone_group": {
          "name": "privatednszonegroup",
          "private_dns_zone_ids": [
            "${azurerm_private_dns_zone.private-dns-zone.id}"
          ]
        },
        "private_service_connection": {
          "name": "appservice-privateconnection",
          "private_connection_resource_id": "${azurerm_app_service.app-service.id}",
          "subresource_names": [
            "sites"
          ]
        },
```

dependencies installed (package.json):
```
  "dependencies": {
    "@cdktf/provider-azurerm": "^0.3.74",
    "cdktf": "^0.8.1",
    "constructs": "^10.0.12"
  },
```

**Comment by DanielMSchmidt** [comment]
This was fixed in cdktf 0.8.6 I believe. A new version of this provider was published, an update should fix the issue for you. If not, please reopen :) 

(Sorry for the month delay in answering, we missed the issues on prebuilt providers)

**Comment by gdamjan** [comment]
seems to work. thanks.
